gwas_sumstats_tools/utils.py

```
@@ -164,21 +164,27 @@ def update_dict_if_not_set(data_dict: dict, field: str, value: any) -> dict:
 
 def split_fields_on_delimiter(data_dict: dict,
                               fields: tuple,
-                              delimiter: str = "|") -> dict:
+                              delimiter: str = "|,") -> dict:
```

Ruff (0.14.14)

[warning] Unused function argument: `delimiter`

(ARG001)